source/index.js

```diff
@@ -36,15 +37,43 @@ var likely = function (node, options) {
 };
 
 /**
+ * Deprecated. Will be removed in 3.0
+ * @returns {Likely}
+ */
+var likely = function () {
+    // eslint-disable-next-line no-console
+    console.warn('This function is DEPRECATED and will be removed in 3.0. Use likely.initiate instead.');
```

Comment:
One more point: it’s not clear from the warning which exact function is deprecated, and finding the right one can be tough. Let’s make the function obvious?